src/main/java/at/ac/tuwien/damap/conversion/AbstractTemplateExportScienceEuropeComponents.java

```diff
@@ -388,69 +387,97 @@ public void dataQuality(){
                 addReplacement(replacements,"[dataorganisation]", dmp.getStructure());
             }
         }
+
```

Comment:
Could store `dmp.getDataQuality()` in a variable to shorten further access and make it more readable

Reply:
generally restructured to be in line with the other refactorings, now has not so many occurrences.